package.json

```diff
@@ -42,6 +41,7 @@
     "@ethersproject/abi": "5.4.1",
     "@ethersproject/bytes": "5.4.0",
     "@ethersproject/providers": "5.4.5",
+    "@zero-tech/zauction-sdk": "0.0.24",
```

Comment:
This should not be a dependency. It should be a `devDependency` and then also a `peerDependency`